libs/rsk/src/lib/rsk.spec.ts

```diff
@@ -0,0 +1,179 @@
+import { rest } from 'msw'
```

Comment:
I believe, according to [Eiki's comment](https://islandis.slack.com/archives/C0134BF9WG6/p1615291957054200), this client should under `libs/clients/rsk`. We would need to move `vmst` client to follow the folder convention in another PR

Reply:
Wasn't sure how this was to be implemented I moved the `rsk` client into a folder `libs/clients`